src/adapter/src/coord/read_policy.rs

```diff
@@ -281,125 +404,133 @@ impl<S: Append + 'static> crate::coord::Coordinator<S> {
             .await
             .unwrap();
         // Update COMPUTE read policies
-        for (compute_instance, compute_ids) in read_holds.id_bundle.compute_ids.iter() {
+        for (compute_instance, compute_ids) in read_holds.compute_ids() {
             let mut policy_changes = Vec::new();
             let mut compute = self.controller.active_compute();
-            for id in compute_ids.iter() {
-                let collection = compute.collection(*compute_instance, *id).unwrap();
-                assert!(collection
```

Comment:
@philip-stoev This panic was removed and the one right above it. The idea being that previously if we were unable to acquire a read hold at a specific time then we would panic. Now instead of panicking we just acquire a hold a the lowest possible time.

I don't know how/if it was ever possible to fail to acquire a read hold to trigger this panic, but at least the API is much more flexible now. I think the idea is that one day storage/compute may have additional constraints on what the initial `since` of a collection can be, and those constraints make the initial `since` larger than the current global read timestamp.